pkg/cmd/etcd.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"go.etcd.io/etcd/clientv3"
+)
+
+type Etcd interface {
+	Put(ctx context.Context, key string, value string) error
+
+	// TODO: implement watcher
+	Watch(ctx context.Context, key string)
+}
```

Comment:
also interfaces should be defined on consumer side, lets move definition of the ConfigStore to Call repo